frontend/src/containers/ReportStep/DoraMetricsChart/index.tsx

```diff
@@ -148,7 +157,39 @@ function extractedMeanTimeToRecoveryDataData(allDateRanges: string[], mappedData
   };
 }
 
-export const DoraMetricsChart = ({ data, dateRanges }: DoraMetricsChartProps) => {
+function isDoraMetricsChartFinish({
+  dateRangeLength,
+  mappedData,
+  type,
+}: {
+  dateRangeLength: number;
+  mappedData: (
+    | ReportResponse
+    | {
+        deploymentFrequencyList: { id: number; name: string; valueList: { value: string }[] }[];
```

Comment:
Think: replace those `key` with the enum `DORAMetricsChartType`

Reply:
done